pyam/core.py

```diff
@@ -254,8 +267,8 @@ def append(self, other, ignore_meta_conflict=False, inplace=False,
                **kwargs):
         """Append any IamDataFrame-like object to this object
 
-        Columns in `other.meta` that are not in `self.meta` are always merged,
-        duplicate region-variable-unit-year rows raise a ValueError.
+        Columns in `other.meta` that are not in `self.meta` are merged.
```

Comment:
Is this meant to be columns or rows? i.e. would new columns be added (and missing values filled with nan), or are new rows added for new scenarios etc.?

Reply:
The result is a "complete" meta dataframe (one row for each model-scenario, one column per indicator, missing values set to nan). I rewrote the docstring for clarity.

Reply:
beauty thanks